### ISSUE TITLE
How to change font size in tables with HTML2PDF ??

### ISSUE THREAD

**Issue by that-ben** [spipu/html2pdf#626]
Just how the heck can I adjust the font-size on a table with this module? I tried ALL sorts of tricks and NOTHING works.  Even setting the whole page to another font size does not affect text in tables! This is driving me nuts.

**Comment by thierryler** [comment]
You have to set the size to the lower level o the cell

```
<style>
.lala {
  font-size: 18pt;
}
</style>

...

<table>
  <tr>
    <td>
      <span class="lala">Lorem ipsum</td>
```

**Comment by that-ben** [comment]
Yes thanks, I already tried that and unfortunately, it does not work.  All the text remains all the same size and it's too big to fit all my reports columns.  I am using an older version, because I couldn't figure out how to simply include HTML2PDF in my PHP app without all this Composer nonsense.  The version I have is simple.  You download the ZIP, put it in a folder and include the main PHP file.  I don't understand all this Composer mess, it's so complicated for nothing.

**Comment by thierryler** [comment]
Did you set a width to each cell? You have to! Sometime I have bugs because of missing widths.

**Comment by that-ben** [comment]
Thanks for the suggestion, but I just tried it and it does not work either.  Still the same font size everywhere.  I have tried all these 3 ways of forcing the TD width:
<td width="100">
<td style="width:100px;">
<style>TD { width:100px; }</style>

**Comment by thierryler** [comment]
Try to add a DIV into your TD, and put the css style onto the DIV.

I have lot of problems with tables...

**Comment by that-ben** [comment]
Thanks again, but no dice sorry.  Thanks for trying tough.  I tried with a simple TD, with a SPAN inside a TD and with a DIV inside a TD.

**Comment by thierryler** [comment]
Maybe show us your code... I have no mode idea :-(
